src/components/ImagesSlider/ImagesSlider.tsx

```diff
@@ -47,13 +47,21 @@ const ImagesSlider: FC<Props> = ({images, onPressHandler}) => {
         style={styles.scrollWrapper}>
         {images.map(({id, src}) => (
           <View key={id} style={[styles.slide, {width}]}>
-            <Pressable onPress={() => onPressHandler(id)}>
+            {onPressHandler ? (
+              <Pressable onPress={() => onPressHandler(id)}>
+                <Image
+                  key={id}
+                  source={src as ImageSourcePropType}
+                  style={styles.image}
+                />
+              </Pressable>
+            ) : (
               <Image
                 key={id}
                 source={src as ImageSourcePropType}
                 style={styles.image}
               />
-            </Pressable>
+            )}
```

Comment:
May be it will be better to put `Image` to the variable to reduce the code?
```js
const image =  <Image key={id} source={src as ImageSourcePropType} style={styles.image} />

onPressHandler ? <Pressable onPress={() => onPressHandler(id)}>{image}</Pressable> : image
```